docs/user_guide/assessment.rst

```diff
@@ -216,6 +216,72 @@ case we see that the subgroup ``(a, 8)`` has a result of ``NaN``, indicating
 that there were no samples in it.
 
 
+.. _control_features_metrics:
+
+Control features for grouped metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Control features (sometimes called 'conditional' features) also divide the input
+data into subgroups. The difference is that aggregations (such as
+:code:`MetricFrame.group_max`) are performed for each subgroup in the conditional
+feature(s), rather than across them (as happens with the sensitive features).
+
+To make this more concrete, consider a loan scenario. In addition to sensitive
+features such as gender and race, applicants can be placed into income bands.
+In this scenario, disparity based on income band is to be expected, but within
+each income band we would want to ensure that there is no disparity across the
+sensitive features. To do this, we make the income band into a control feature
+(in the statistical sense of 'controlling for a variable').
+
+The :class:`MetricFrame` constructor allows us to specify control features in
+a manner similar to sensitive features, using a :code:`conditional_features=`
+parameter:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> income_band = ['l','l','h','l','h','l','l','h','h','l','l','h','h','l','l','h']
+    >>> metric_c_f = MetricFrame(skm.precision_score,
+    ...                          Y_true, Y_pred,
+    ...                          sensitive_features=g_2,
```

Comment:
Can we please change this to gender and add a comment saying gender has two values female and male?

Reply:
If we use gender we should include non-binary.

Reply:
I think there are two points here

1. Sex itself is not binary, yet pretty much all datasets out there with a _sex_ attribute treat is as binary, usually with _female_, _male_ values as options as opposed to more accepted terms _woman_ and _man_ used for gender, but it really isn't binary (https://massivesci.com/articles/sex-gender-intersex-transgender-identity-discrimination-title-ix/) 
2. Sex and gender can mean very different things (https://en.wikipedia.org/wiki/Sex_and_gender_distinction). Hypothetically you could even have a dataset with both fields and have them not be the same. I'm not sure what it is that this dataset is measuring and how they got the data, so I would double check that before just switching the terms.

disclaimer: this is not a topic which I find the easiest to navigate, and I wouldn't be surprised if I've made a few mistakes in this comment, and if that's the case, I'm happy to hear what they are and apologize.

Reply:
This particular dataset is measuring my ability to get a metric with a different value for each entry.... I think that's why in the rest of the doc, we steered away from concrete feature names and classes.

Reply:
@riedgar-ms This example isn't about abstract metrics and values and entities, it says:

> ...high income females should be treated similarly to high income males and low income females should be treated similarly to low income males.

That's one example of a normative decision.  You are describing *how you think the world ought to be*.

@mesameki makes another normative decision in writing:

> gender has two values female and male

If this is a statement about people, it's incorrect.  If it's a statement about the dataset, it's a normative decision to construct the dataset in that way.

Separately, the writing signals something to people reading this as well; gender has been the subject of extensive research in the ML fairness literature and in HCI (eg, [Algorithmically-Encoded Identities](https://alexhanna.github.io/algo-identity/) from FAT* 2020, [HCI Guidelines for Gender Equity and Inclusivity](https://www.morgan-klaus.com/gender-guidelines.html)).  I really ❤️ the [choose-your-own-adventure style narrative](https://github.com/alexhanna/algo-identity/blob/master/choose-your-own-administrative-violence-adventure/full.pdf) for illustrating why this matters to people.

I hope that helps you all decide what you think is best! 😄

Reply:
@kevinrobinson Fair point. I have rephrased to make the example more abstract.

Reply:
@kevinrobinson  that makes a lot of sense. I meant in this dataset. But I get your point and agree.